pkg/util/container.go

```diff
@@ -324,5 +324,5 @@ func ParseContainerId(data string) (cType, cID string, err error) {
 
 // WriteCgroupCPUSet writes the cgroup cpuset file according to the specified cgroup dir
 func WriteCgroupCPUSet(cgroupFileDir, cpusetStr string) error {
```

Comment:
consider setting cgroup file writing methods as private in resourceexecutor pkg, in case that other pkg use directly.

Reply:
marked as DEPRECATED. also refactor the caller CPUSuppress to adapt the new cgroup updater.